engines/dfc_provider/app/controllers/dfc_provider/base_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 # Controller used to provide the API products for the DFC application
 module DfcProvider
-  class BaseController < ActionController::Base
+  class BaseController < ApplicationController
```

Comment:
No, this is the base for API controllers and they shouldn't have all the bloat of the ApplicationController. I think that the better solution is to rename this BaseController to ApplicationController within this module here. That should satisfy Rubocop.